docs/conf/prague/2021/news/tickets-cfp-reminder.rst

```diff
@@ -0,0 +1,66 @@
+=======================================================
+
+Greetings, documentarians! Today we're happy to announce that tickets are `on sale now <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_, and to remind you that our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_ is still open.
+
+Following the success of the 2020 virtual conference, we look forward to more opportunities to bring the community together virtually and to share the wonderful energy with each other wherever you are in the world.
+
+If there is something you’d really like to see a talk about this year, submit a proposal for it or refer someone else who would be good!
+
+Get your ticket for the conference
+----------------------------------
+
+**The Prague conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/prague/{{year}}/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer.
+
+If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_! Even though our capacity is more flexible due to the virtual event platform, we might still sell out to make sure we're providing a manageable event size for our attendees, as always.
+
+Similar to last year's virtual conference, we're going to keep as much as possible of the spirit of the event alive:
+
+* Excellent single track of pre-recorded presentations with live moderated Q&A after each talk
+* Writing Day sessions
+* Unconference sessions
+* Job Fair booths
+* Sponsor booths
+* A virtual hallway track
+* A social event of some kind, bring your own beverage and snacks :)
+
+We really want to make sure that folks can get value from being at the event at the same time, and we think it will be another great year of helping our community connect.
+
+Submit your talk by June 30th
+-----------------------------
+
+There are still a few weeks left to submit your proposal! Check out the `Call for Proposals page <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_ for more details about what we are looking for.
+
+And if you already have an idea but you’re having a hard time getting your proposal to come together, you can always ask for feedback on the `Write the Docs Slack <https://www.writethedocs.org/slack/>`_. The community is there to help!
+
+We’ll then be reviewing all the talk proposals, and will be contacting everyone who submitted a proposal by **{{ cfp.notification }}**.
+
+Grant program
+-------------
+
+We updated our grant program application to take into account the virtual format of the event.
+We're excited to be able to offer free tickets to a larger number of people, since there are now minimal other costs to attend the event.
+
+You can apply by **{{ grants.ends }}** on `our website <https://www.writethedocs.org/conf/prague/{{year}}/opportunity-grants/>`_.
```

Comment:
grants.ends is not in the config yet, that's why the build breaks - I will add that a bit later